console/src/components/Common/TableCommon/DragFoldTable.js

```diff
@@ -87,7 +90,11 @@ class DragFoldTable extends Component {
             })
           }
           folded={this.state.folded}
+          showPagination={!this.props.countLoading}
```

Comment:
Is this needed?

Reply:
When the count is still loading there's no pagination displayed. Instead, there's a spinner.